awesomerc.lua

```diff
@@ -175,6 +174,14 @@ awful.screen.connect_for_each_screen(function(s)
         end
         gears.wallpaper.maximized(wallpaper, s, true)
     end
+end
+
+-- Re-set wallpaper when a screen's geometry changes (e.g. different resolution)
+screen.connect_signal("property::geometry", set_wallpaper)
```

Comment:
Should we add a "connect_for_each_screen_change" into `awful.screen`? It seems like it will become a common use case.